tests/common/plugins/tacacs.py

```diff
@@ -47,11 +56,13 @@ def cleanup_tacacs(ptfhost, duthost, tacacs_server_ip):
 @pytest.fixture(scope="module")
 def test_tacacs(ptfhost, duthost, creds):
     tacacs_server_ip = ptfhost.host.options['inventory_manager'].get_host(ptfhost.hostname).vars['ansible_host']
-    configure_tacacs(ptfhost, duthost, creds, tacacs_server_ip)
+    config_tacacs_on_switch = request.config.getoption('--switch_tacacs_config')
```

Comment:
if the user does not provide switch_tacas_config is it false?

Reply:
Yes, it's False by default